UI-III-Flexbox/great-idea/services.html

```diff
@@ -0,0 +1,132 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+
+  <title>Great Idea!</title>
+  
+  <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
+  <link rel="stylesheet" href="./index-UI3.css">
+
+  <!--[if lt IE 9]>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
+  <![endif]-->
+</head>
+
+<body>
+    <div class="container">
+    <div class="website-wrapper">
+     <div class="static-navbar">
+		<nav>
+			<a href="services.html">Services</a>
+			<a href="#">Product</a>
+			<a href="#">Vision</a>
+			<a href="#">Features</a>
+			<a href="#">About</a>
+			<a href="#">Contact</a>
+			<div class="top-logo">
+				<img class="logo" src="https://github.com/LambdaSchool/User-Interface/blob/master/great-idea-website/img/logo.png?raw=true" alt="Great Idea! Company logo.">
+			</div>
+		</nav>
+	</div>
+
```

Comment:
remember to try and have your indents lined up so its easier to tell where things begin and end